src/commands/auth/jwt/grant.ts

```diff
@@ -64,7 +64,7 @@ export default class Grant extends SfdxCommand {
   public async run(): Promise<AuthFields> {
     let result: AuthFields = {};
 
-    if (await Prompts.shouldExitCommand(this.ux, this.flags.noprompt)) return {};
+    if (await Prompts.shouldExitCommand(this.ux, this.flags.noprompt as boolean)) return {};
```

Comment:
**Note**: `this.flags.noprompt` will be `undefined` unless the flag is specified (it isn't required). Seems ok as `undefined` is a falsey value.